SwiftDeck/SwiftDeckExamples/ViewController.swift

```diff
@@ -12,19 +12,21 @@ import UIKit
 final class ViewController: UIViewController {
```

Comment:
Following up on previous discussion. In other language bindings, we would call these directories `swiftdeck/src` and `swiftdeck/examples` but I assume this is following conventions in the swift world.